src/Export/statdesc.lua

```diff
@@ -69,6 +69,10 @@ function loadStatFile(fileName)
 						end
 						table.insert(desc.limit, limit)
 					end
+					-- Manual fix for GGG typo
```

Comment:
Could you report this error? Maybe we'll be able to remove this workaround in the future then.

Reply:
Yeah, I can.  Not sure how it's working in-game, since the Pure Talent tooltip shows up fine, so I wasn't sure if they'll bother to fix it.